dotCMS/src/main/java/com/dotcms/security/apps/AppsAPIImpl.java

```diff
@@ -310,7 +311,7 @@ public void deleteSecrets(final String serviceKey, final Host host, final User u
     private static final String DESCRIPTORS_MAPPED_BY_SERVICE_KEY = "DESCRIPTORS_MAPPED_BY_SERVICE_KEY";
 
     @Override
-    public List<ServiceDescriptor> getServiceDescriptors(User user)
+    public List<AppDescriptor> getAppDescriptors(User user)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'user' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5218098)